app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
+  before_action :is_signed_in?, only: [:new, :create]
+
+  def new
+    @post = Post.new()
+  end
+
+  def create
+    post = Post.new(title: params[:post][:title], body: params[:post][:body], user_id: current_user.id)
```

Comment:
You could improve this by creating the **private** `post_params` method, and changing this line after that to be
`@post = current_user.posts.build(post_params)`